data/json/mapgen/animalshelter.json

```diff
@@ -0,0 +1,271 @@
+[
+  {
+    "id": "animalshelter",
+    "type": "vehicle_group",
+    "vehicles": [
+      [ "animalctrl", 300 ],
+      [ "cube_van", 100 ],
+      [ "cube_van_cheap", 50 ]
+    ]
+  },
+  {
+    "id": "animalshelter_softdrug",
+    "type": "item_group",
+    "items": [
+      [ "bandages", 50 ],
+      [ "saline", 20 ],
+      [ "vitamins", 15 ],
+      [ "calcium_tablet", 15 ],
+      [ "nyquil", 5 ],
+      [ "disinfectant", 15 ],
+      [ "elec_hairtrimmer", 15 ],
+      [ "survnote", 5 ],
```

Comment:
Also curious on why you added "survivor's note" to item_group "animalshelter_softdrug". These survivor's notes have various flavor texts on them, many of which I'd imagine being out of place finding them with various medical supplies in an animal shelter.

Reply:
>Also curious on why you added "survivor's note" to item_group "animalshelter_softdrug". These survivor's notes have various flavor texts on them, many of which I'd imagine being out of place finding them with various medical supplies in an animal shelter.

Two reasons I had them was to help explain the lack of locked up animals (someone had to free them, and maybe it wasn't always a worker or maybe a worker was a survivor) and the lack of medical supplies (someone might've looted the place --- so these notes could be just little clues to that).

Reply:
>And to make it so that not all of them are open and to add a bit of randomness to the location I would personally do something like this...
"^": [ "t_chaingate_c", "t_chaingate_o", "t_chaingate_o" ],
Which would result in a 2/3 chance of the gates randomly being open. And...
"+": [ "t_door_c", "t_door_o" ],
so the front doors will have a 50% chance of being left open.

Ah, see I didn't realize the small little techniques like this. That's great to know for future reference.